site/viamguides/src/rgbled/rgbled.md

```diff
@@ -0,0 +1,274 @@
+author: Joyce
+id: rgbled
+summary: Control an RGB LED with a Raspberry Pi
+categories: Getting-Started, Developer
+environments: web
+status: Published
+feedback link: https://github.com/viam-labs/viamcodelabs/issues
+tags: Getting Started, Developer, Data
+
+# Control an RGB LED with a Raspberry Pi
+
+<!-- ------------------------ -->
+
+## Overview
+
+Duration: 1
+
+[RGB LEDs](https://en.wikipedia.org/wiki/Light-emitting_diode#RGB_systems) are light-emitting diodes that mix red, green, and blue to create many different hues of light. In this codelab, learn how to control an RGB LED with your Raspberry Pi. Whether you're building a fun alert system or adding light effects to a DIY project, you can integrate a RGB LEDs into your Viam project quickly using a prebuilt module.
+
+<img src="assets/brightLED.png" alt="close up view of RGB LED" width="300">
+
+### What You’ll Build
+
+- A common cathode RGB LED that can display a specific color or light effect.
+
+<img src="assets/blueMushroom.png" alt="close up view of RGB LED" width="300">
+
+### Prerequisites
+
+- A computer with MacOS, Windows, or Linux to flash your Raspberry Pi and configure the device's components using the Viam app
+- Hardware and supplies requirements
+
+  - 1 - [Raspberry Pi 4](https://a.co/d/fc3JUMm)
+  - 1 - microSD card to use with your Pi
+  - 1 - power supply for your Pi
+  - 1 - [common cathode RGB LED](https://a.co/d/inuxrYr) with 3 resistors
```

Comment:
What level of resistance for the resistors? If the Ohm level is too high then no power will get to the LED. 

You can figure out which ones you're using with this handy too: https://www.digikey.com/en/resources/conversion-calculators/conversion-calculator-resistor-color-code